ptf_nn/ptf_nn_agent.py

```diff
@@ -135,6 +136,7 @@ def __init__(self, dev, port, iface_name):
         self.dev = dev
         self.port = port
         self.iface_name = iface_name
+        os.system("ifconfig %s up" % self.iface_name)
```

Comment:
would you mind using the `subprocess` module instead here. Please see https://docs.python.org/2/library/subprocess.html#subprocess-replacements